example/android/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
+        android:name="${applicationName}"
```

Comment:
Just checking this is defined & we haven't missed defining it?

Ideally I should add some CI tests to check app boots and lib works on app etc

Reply:
Should I revert this change ?

Reply:
> Just checking this is defined & we haven't missed defining it?
> 
> Ideally I should add some CI tests to check app boots and lib works on app etc

If we replace `"${applicationName}"` with `"io.flutter.app.FlutterApplication"` it will throw error like

```Your Flutter application is created using an older version of the Android
embedding. It is being deprecated in favor of Android embedding v2. Follow the
steps at
https://flutter.dev/go/android-project-migration
to migrate your project. You may also pass the --ignore-deprecation flag to
ignore this check and continue with the deprecated v1 embedding. However,
the v1 Android embedding will be removed in future versions of Flutter.```

so we have to use "${applicationName}"